docs/background/releases.rst

```diff
@@ -452,7 +452,7 @@ Features
   device model name and unique ID e.g. a Pixel 3a shows up as ``Pixel 3a
   (adbDeviceId)``. (#433)
 * Android apps are now packaged in Android App Bundle format. This allows the
-  Play Store to dynmically build the smallest APK appropriate to a device
+  Play Store to dynamically build the smallest APK appropriate to a device
```

Comment:
Side note - I'd *love* to get a spell checker in as part of the linting pass...

Reply:
I agree; however, when I ran (PyCharm's) spellcheck on the docs, at least 90% of the results were false-positives...

Reply:
[`sphinxcontrib.spelling`](https://sphinxcontrib-spelling.readthedocs.io/en/latest/) allows for custom dictionaries...

Reply:
Spellchecking is now live.